microsoft/vscode
# Corrected the regex to detect espaced percent symbol.

**This PR fixes #138921. The fix is correcting the regex that was being used for inline comment.**


"%" should comment an inline comment but "\%" should escape and treat the text if you write % along with a text after it. Example:

$$ % This is comment $$ 
Rendered as empty space.

$$ \% Show this text $$
Rendered as "%Showthistext" 

**How to test**

- You can refer to this file to see how VS renders it: /vscode/extensions/vscode-colorize-tests/test/colorize-fixtures/md-math.md
- You can create your own .md file and test the cases.
